akinyeleolat/politico
# #163498816 setup dev environment

#### What does this PR do?
Set up development environment
#### Description of Task to be completed?

- Eslint, morgan, babel and other dependencies and development dependencies were added
- Express was added as a dependency
- Lastly, I added app.js 

#### How should this be manually tested?
- Clone repo
- npm install
- npm start and then 
- open a browser and enter "http://localhost:5000", you should see a welcome message
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
163498816
#### Screenshots (if appropriate)
#### Questions:
